tests/test_domain_objects/cash_flow_test.py

```diff
@@ -0,0 +1,58 @@
+import sys
+from datetime import datetime, timezone
+
+sys.path.insert(0, 'tests/')
+from utils import shared_tests as shared
+from utils import helper_methods as helper
+
+
+def test_cash_flows():
+    records, domain_obj = helper.set_up_cash_flow_tests()
+    for record in records:
+        shared.attribute_quantity_valid('cash_flow', record, 7)
+        account_id_valid(record, domain_obj)
+        corporate_action_id_valid(record)
+        quantity_valid(record)
```

Comment:
Using the shared `price_valid` method will also check if it is a float to 2 dp:

`shared.price_valid(record, field_name='quantity', min=1, max=10000)`

Remove the `quantity_valid` method from this test